common/src/main/scala/explore/undo/UndoContext.scala

```diff
@@ -76,7 +76,7 @@ case class UndoContext[M](
       _ <- push(undoStack)(Restorer[DefaultA, M, A](model.get, getter, setter, onRestore))
       _ <- reset(redoStack)
       _ <- model.mod.compose(setter)(v)
-      f <- onSet(model.get, v).runAsync
+      f <- onSet(model.get, v).runAsyncAndForget // TODO: Log errors
```

Comment:
Does `runAsync` not have the same behaviour, but log errors?

Reply:
Yes, but we need a `Logger` for that and we are now creating a global `UndoContext` from the router, which doesn't have a `Logger` instance on hand.

I'll have to look into how to get a `Logger` there, or else modify the `UndoContext` so that the `Logger` is required by the methods instead of the constructor.

Future PR, for now I put this note in the code.